flutter/documentation/result-spec.md

```diff
@@ -61,10 +61,13 @@ If you enable Submission mode, both `performance_run` and `accuracy_run` values
   * `throughput`: floating point number  
     Throughput value for this run of the benchmark.
     May be null for an accuracy run.
-  * `accuracy`: floating point number  
-    Accuracy value for this run of the benchmark.
-    Value must be normalized between `0` and `100`.
+  * `accuracy`: map
     May be null for a performance run if groundtruth file is not provided.
+    * `value`: floating point number  
```

Comment:
I don't think that being normalized is a defining property of this field. It's not like we have `normalizedValue` and `nonNormalizedValue` fields, there is only one numeric field.
`string` value for accuracy is not a "value that may not be normalized", it's a human-readable string value. Normalization isn't even applicable to this field because it's non-numeric.

> I also think it's more common to normalize the value between `0.0` and `1.0`.

Ok, seems like a good idea to me.

Reply:
> I also think it's more common to normalize the value between `0.0` and `1.0`.

Done

Reply:
> Accuracy value for this run of the benchmark.
       Value must be normalized between `0.0` and `1.0`.

The idea is to know that we use normalized value without reading the specs.

Reply:
> I would explicitly name this `normalizedValue`

Done.

Though, I still don't see why this would be useful.